web/src/components/RestaurantSetting/RestaurantSetting.js

```diff
@@ -0,0 +1,150 @@
+import useOnClickOutside from '../../hooks/useOnClickOutside'
+
+import { Form, Label, TextField, FieldError, Submit } from '@redwoodjs/forms'
+import { useMutation, useQuery } from '@redwoodjs/web'
+import { useForm } from 'react-hook-form'
+import { useState, useRef, useEffect } from 'react'
+import { useAuth } from '@redwoodjs/auth'
+import { toast, Toaster } from '@redwoodjs/web/toast'
+
+import { SketchPicker } from 'react-color'
+
+const UPDATE_RESTAURANT = gql`
+  mutation UpdateRestaurantMutation(
+    $id: String!
+    $input: UpdateRestaurantInput!
+  ) {
+    updateRestaurant(id: $id, input: $input) {
+      id
+    }
+  }
+`
+
+const GET_RESTAURANT = gql`
+  query GetRestaurant($ownerId: String!) {
+    restaurantByOwnerId(ownerId: $ownerId) {
+      id
+      name
+      brandColor
+    }
+  }
+`
+
+const RestaurantSetting = () => {
```

Comment:
Same as above, I'd like to rename `RestaurantSetting` to `RestaurantSettings`. 👍🏻